CountryPicker/CountryPickerTests/CountryPickerViewTests.swift

```diff
@@ -94,6 +94,13 @@ class CountryPickerViewTests: XCTestCase {
         XCTAssertEqual(numberOfRows, 1)
     }
     
+    func test_setSelectedCountry() {
+        let sut = makeSUT()
+        XCTAssertEqual(sut.dataSource?.pickerView(sut, numberOfRowsInComponent: 0), 2)
+        sut.setSelectedCountry(Country(countryCode: "US"))
+        XCTAssertEqual(makeSUT(selectedCountry: countries[1]).selectedRow(inComponent: 0), 1)
```

Comment:
why you have create multiple sut ?
we should match country to test if selected country is the one which is selected in ui

Reply:
@SURYAKANTSHARMA  line 101 tests if the picker actually selects the right row on setting `setSelectedCountry(Country(countryCode: "US"))`

Reply:
ya but why you are creating new sut ? makeSUT(selectedCountry: countries[1]) ?
and can we make it more readable something like 
let selectedCountry = Country(countryCode: "US")
sut.setSelectedCountry(selectedCountry)
XCTAssertEqual(sut.selectedCountryInUI, selectedCountry )

Reply:
@SURYAKANTSHARMA Makes sense, Thanks for  your comment. Updated as mentioned. Please let me know